tests/integration/microprofile/rest-client/pom.xml

```diff
@@ -89,7 +94,20 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.maven.surefire</groupId>
+                        <artifactId>surefire-junit47</artifactId>
+                        <version>${surefire.version}</version>
```

Comment:
This should probably be in `dependencyManagement` section?

Reply:
Are you sure? You would force every module using the surefire plugin to have these dependencies, too.

Reply:
No, I mean the version management, not plugin management

Reply:
I see. That's a good point